dashboard/src/t5gweb/database/session.py

```diff
@@ -0,0 +1,67 @@
+"""Database session and connection management"""
+
+import threading
+from typing import Optional
+
+from sqlalchemy import URL, create_engine
+from sqlalchemy.orm import DeclarativeBase, sessionmaker
+from t5gweb.utils import set_cfg
+
+cfg = set_cfg()
+
+
+def get_database_url():
+    """Build database URL from configuration"""
+    url_object = URL.create(
+        "postgresql",
+        username=cfg["postgresql_username"],
```

Comment:
the postgresl credentials will be stored as a secret in OpenShift. you'll want to pull them from the environment rather than the config file, similar to the secret for JIRA.